suite-native/module-staking-management/src/components/StakingBalancesOverviewCard.tsx

```diff
@@ -132,8 +159,27 @@ export const StakingBalancesOverviewCard = ({
                     <Text color="textSubdued">
                         <Translation id="staking.apy" />
                     </Text>
-                    <Text>{apy}%</Text>
+                    <Text>
+                        {!isAdaStakedWithFiveBinaries ? (
+                            `${apy}%`
+                        ) : (
+                            <Translation id="staking.notAvailable" />
+                        )}
+                    </Text>
                 </Box>
+
+                {symbol === 'ada' && (
+                    <Box paddingTop="sp16">
+                        <Box style={applyStyle(separatorStyle)} />
+
+                        <Box style={applyStyle(infoItemStyle)}>
+                            <Icon name="info" color="textSubdued" size="mediumLarge" />
+                            <Text color="textSubdued">
```

Comment:
according to the designs it should be a `hint` variant 
<img width="1730" height="658" alt="Screenshot 2025-12-17 at 15 29 43" src="https://github.com/user-attachments/assets/1edb3c79-d9e3-4786-8e02-3d940d15745f" />